### PR TITLE
Frontend sprint

### PR DESCRIPTION
**Pull request by matthew-jugovic**
Merging frontend work into main

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # Created by https://www.toptal.com/developers/gitignore/api/node,windows,macos
 # Edit at https://www.toptal.com/developers/gitignore?templates=node,windows,macos
 
+
+### replay service ###
```
Comment (chrishollandaise) [inline .gitignore:5]: duplicate

File: webapp/src/components/visualizations/Replay2DView/Replay2DView.tsx
Comment (chrishollandaise) [inline webapp/src/components/visualizations/Replay2DView/Replay2DView.tsx:1]: I'm leaving a note here. Looks like it's using an older version here? Also, reminder for myself that players must be dynamically loaded into the ECS

File: webapp/src/components/general/header.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
+import logo from "../../assets/logo/72.png"
+import { useContext } from "react"
+import { HeaderContext, UserContext } from "../../context/contexts"
+
+// export default function About() {
```
Comment (chrishollandaise) [inline webapp/src/components/general/header.tsx:5]: This should be removed if you are not using it

File: webapp/src/assets/images/backgrounds/stats_general.png
Comment (chrishollandaise) [inline webapp/src/assets/images/backgrounds/stats_general.png:1]: I notice there are duplicate backgrounds. Is this intended?

**Review by cdaise — COMMENTED** [review]
lgtm, I left some comments. You can choose to make the changes or don't

**Review by cdaise — APPROVED** [review]

**Pull request merged**